### PR TITLE
feat(component-status): initial commit of component-status

### LINKED ISSUE #124: [Enhancement] Component Status in Cookbook
We need to have a "Component Status" page in the cookbook, that shows the status for each component: UX (Version + Link to Zeplin) | UI | Code | Anything else??
See https://polaris.shopify.com/components/component-status and https://clarity.design/documentation for inspiration.

### PR DESCRIPTION
**Pull request by hafstad**
closes #124

### CODE REVIEW

**Comment by hafstad** [comment]
I've started the component status, this is the initial state which can easily be improved / extended with links to zeplin etc.:
image

File: src/app/app.module.ts [hunk 1]
```diff
@@ -10,10 +10,18 @@ import { SideNavComponent } from './page/side-nav/side-nav.component';
...
 @NgModule({
   declarations: [AppComponent, HomeComponent, SideNavComponent, HeaderComponent, IntroComponent],
-  imports: [BrowserModule, AppRoutingModule, HttpClientModule, FormsModule, KirbyModule],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    HttpClientModule,
+    FormsModule,
+    KirbyModule,
+    ComponentStatusModule,
```
Comment (jakobe) [inline src/app/app.module.ts]: Shouldn't need to import `ComponentStatusModule` as it's lazy loaded
```suggestion
```

File: src/app/component-status/component-status-items.ts [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+}
+
+enum ItemUXAndDesignStatus {
+  underConsideration = 'Under consideration',
+  notCurrentlyPlanned = 'Not currently planned',
+  inProgress = 'In progress',
+  done = 'Done',
+}
+
+export const componentStatusItems: ComponentStatusItem[] = [
```
Comment (jakobe) [inline src/app/component-status/component-status-items.ts]: We should probably sync this with Designsystem 1.0 from the UX'ers

File: src/app/component-status/component-status.component.tns.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<ScrollView>
```
Comment (jakobe) [inline src/app/component-status/component-status.component.tns.html]: Don't think we wanna keep a {N} version of the component status

File: src/app/component-status/component-status.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { componentStatusItems, ComponentStatusItem } from './component-status-items';
+
+@Component({
+  selector: 'kirby-component-status',
+  templateUrl: './component-status.component.html',
+  styleUrls: ['./component-status.component.scss'],
+})
+export class ComponentStatusComponent implements OnInit {
```
Comment (jakobe) [inline src/app/component-status/component-status.component.ts]: Don't think we wanna keep a {N} version of the component status

**Pull request closed without merging**